html/index.html

```diff
@@ -11,15 +11,33 @@
   </head>
 
   <body>
-    <div id="pageTitle">
-      <h1>Binding Of Isaac: Challenge Generator</h1>
+    <div class="wrapper">
+      <div id="pageTitle">
+        <h1>Binding Of Isaac: Challenge Generator</h1>
+      </div>
+      <div class="challengeTitle"><span id="title"></span></div>
```

Comment:
Is there a reason there is a span with an ID inside this div? Would this not be simpler?

```suggestion
      <div class="challengeTitle" id="title"></div>
```